blockchain/genesis.go

```diff
@@ -95,47 +117,53 @@ func NewGenesisBlock(cfg *config.Config) *Block {
 	if err := yaml.Unmarshal(actionsBytes, &actions); err != nil {
 		logger.Panic().Err(err).Msg("Fail to create genesis block")
 	}
+
+	Gen.CreatorPubKey = actions.Creation.PubKey
+	Gen.CreatorPrivKey = actions.Creation.PriKey
+	creatorPubk, creatorPrik := decodeKey(Gen.CreatorPubKey, Gen.CreatorPrivKey)
+	creatorAddr := Gen.CreatorAddr(cfg.Chain.ID)
+
 	votes := []*action.Vote{}
 	for _, nominator := range actions.SelfNominators {
-		pubk, err := keypair.DecodePublicKey(nominator.PubKey)
+		pk, sk := decodeKey(nominator.PubKey, nominator.PriKey)
+		address := generateAddr(cfg.Chain.ID, pk)
+		vote, err := action.NewVote(
+			0,
```

Comment:
@lizhefeng, would you please comment if 0 nonce is correct or not?

Reply:
This is correct. Meant to be 0.